### PR TITLE
Packagekit: fix some minor issues

### LINKED ISSUE #11222: PackageKit: Applying Updates Failed
I had some DNS failure while trying to update my cockpit 186 install. When it failed I got greeted by a `<pre>` that spans the entire width available and is not offsetted from the header. As shown:

image

There is no clear way to revert back to the updates page.

### LINKED ISSUE #15344: updates: misaligned spinner
Cockpit version 237

Caught in PR #14581 by @garrett:

Spinner in the "installing updates.." view is misaligned and doesn't have proper spacing.

Screenshot_2021-02-10 Software Updates - admin localhost localdomain

### LINKED ISSUE #15353: "No delay" in reboot dialog should be on top
Side issue in PR #14581: No delay should be on top, but 1 minute should still be default.

Screenshot_2021-02-16 Software Updates - admin localhost localdomain(7)

### PR DESCRIPTION
**Pull request by KKoukiou**
Fixes #15344
Fixes #15353
Fixes #11222

New spinner is from PF4: no extra styles are added there; regarding alignment I really can't notice something weird:

Screen Shot 2021-02-18 at 18 23 41

Reboot done as requested:

Screen Shot 2021-02-18 at 18 18 59

New error page:

Screen Shot 2021-02-18 at 18 01 43

or 

Screen Shot 2021-02-18 at 17 58 53

Old error page:

Screen Shot 2021-02-18 at 17 59 25

### CODE REVIEW

File: pkg/packagekit/updates.jsx [hunk 1]
```diff
@@ -1323,7 +1323,24 @@ class OsUpdates extends React.Component {
...
+                <>
+                    <EmptyStatePanel title={ STATE_HEADINGS[this.state.state] }
+                                     icon={ ExclamationCircleIcon }
+                                     paragraph={
+                                         <TextContent>
+                                             <Text component={TextVariants.p}>
+                                                 {this.state.errorMessages.map(m => <span key={m}>{m}</span>)}
+                                             </Text>
+                                             <Text component={TextVariants.p}>
+                                                 {_("Try resolving the issue and then refreshing the page")}
```
Comment (martinpitt) [inline pkg/packagekit/updates.jsx:1337]: Not sure, not a native speaker myself, but: "Please reload the page after resolving the issue."?

**Review by martinpitt — CHANGES_REQUESTED** [review]
Needs test adjustments. Thanks for the fixes!

**Comment by martinpitt** [comment]
Marking release-blocker for 238.1 next Monday, this is good stuff for polishing.

File: pkg/packagekit/updates.jsx (hunk 1 above)
Comment (KKoukiou) [reply pkg/packagekit/updates.jsx]: It's definitely better than what I had. thanks :)

**Review by martinpitt — APPROVED** [review]
Epharisto! LGTM if tests agree.

**Comment by KKoukiou** [comment]
> Epharisto! LGTM if tests agree.

They don't I will reask for review once it's green ;)

**Review by martinpitt — APPROVED** [review]
:green_circle: thanks!

**Pull request merged**